### PR TITLE
Add mvi timeout

### PR DESCRIPTION
**Pull request by kreek**
This could also be a hotfix for prod if need be.
All MVI environments are currently timing out but this allows devops to set the MVI open timeout (MVI_OPEN_TIMEOUT) and read timeout (MVI_TIMEOUT) as they see fit.

### CODE REVIEW

**Comment by patrickvinograd** [comment]
Don't merge this without corresponding devops change to add environment variables.

**Comment by knkski** [comment]
Related to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/423

**Comment by plusjeff** [comment]
LGTM to me. OK to merge when build completes.

File: config/initializers/figaro.rb [hunk 1]
```diff
@@ -12,6 +12,8 @@
   'MHV_SM_APP_TOKEN',
   'EVSS_BASE_URL',
   'MVI_URL',
+  'MVI_OPEN_TIMEOUT',
```
Comment (omgitsbillryan) [inline config/initializers/figaro.rb]: Would it be possible to just default these values to some values in the event the environment variables dont exist?  It seems a little harsh to require them in Figaro.

**Pull request merged**